vnext/Test/WebSocketServer.cpp

```diff
@@ -317,8 +317,8 @@ void SecureWebSocketSession::OnSslHandshake(error_code ec)
 
 #pragma region WebSocketServer
```

Comment:
Wondering if it might make sense to consolidate vnext/Test into one of the other UT/IT projects in vnext.

Reply:
It needs to be consumable by the Desktop UT/IT projects too, plus, it's part of an exported test DLL used in Office.

If the MSRN UT/ITs have a use for this code, we could perhaps turn the project into Shared Items.